src/components/index.ts

```diff
@@ -67,3 +67,5 @@ export * from './utils/setRef';
 export {useOnFocusOutside} from './utils/useOnFocusOutside';
 export * from './utils/xpath';
 export * from './utils/useFileInput/useFileInput';
+export * from './utils/getSrcSet';
+export * from './utils/getClosestNumber';
```

Comment:
Why we export these two?

Reply:
I guess they could be omited, will remove this exports